request.js

```diff
@@ -31,6 +31,7 @@ module.exports = function createClient(auth) {
 		return request(verb, 'https://api.flickr.com/services/rest')
 		.query('method=' + method)
 		.query(args)
+		.set('X-Flickr-API-Method', method)
```

Comment:
@jeremyruppel this header is making me fail all browser requests. I don't understand what it does nor what I should do to make it work in the browser without oauth.

Can you help me?

Reply:
Ah yep, this bit will fail in the browser. I will cut a new release today to remove this, but in the meantime you can use superagent's [Request.unset](https://github.com/visionmedia/superagent/blob/master/lib/request-base.js#L294-L310) to drop this header.

Reply:
Ok, with the unset works but I got another error. I'll open an issue and we see it there. Thanks